README.rst

```diff
@@ -451,7 +451,7 @@ argument is a pointer to the internals of the object the new outlet is
 created for.
 
 The second argument is a symbolic description of the outlet type. Since
-out counter should output numeric values it is of type “float”.
+our counter should output numeric values it is of type “float”.
```

Comment:
and I would **really** appreciate it, if you could split out spelling fixes into separate commits